guides/v2.3/graphql/caching.md

```diff
@@ -0,0 +1,101 @@
+---
+group: graphql
+title: GraphQL caching
+---
+
+Magento can cache pages rendered from the results of certain GraphQL queries with [full-page caching]({{page.baseurl}}/extension-dev-guide/cache/page-caching.html). Full-page caching improves response time and reduces the load on the server. Without caching, each page might need to run blocks of code and run retrieve large amounts of information from the database. Only queries submitted with an HTTP GET operation can be cached. POST queries cannot be cached.
+
+## Cached queries
+
+The definitions for some queries include cache tags. Full page caching uses these tags to keep track of cached content. They also allow public content to be invalidated. Private content invalidation is handled on the client side.
+
+Magento caches the following queries:
+
+* `category`
+* `cmsBlocks`
+* `cmsPage`
+* `products`
+* `urlResolver`
+
+Magento explicitly disallows caching the following queries.
+
+* `cart`
+* `country`
+* `countries`
+* `currency`
```

Comment:
Just out of curiosity, Why wouldn't it be valid to cache  countries, country and currency?